src/plugins/intel_gpu/src/plugin/graph.cpp

```diff
@@ -447,9 +468,19 @@ std::shared_ptr<ngraph::Function> Graph::GetExecGraphInfoByPrimitivesInfo(std::v
     return std::make_shared<ngraph::Function>(results, params, "runtime_gpu_graph");
 }
 
+void Graph::Export(cldnn::BinaryOutputBuffer &ob) {
+    ob << m_program->inputLayouts;
+    ob << primitiveIDs;
+    ob << outputDims;
+
+    auto m_network = m_networks.back();
+
+    m_network->save(ob);
+}
+
 std::shared_ptr<ngraph::Function> Graph::GetExecGraphInfo() {
     auto primitives_info = GetNetwork()->get_primitives_info();
-    return GetExecGraphInfoByPrimitivesInfo(primitives_info, true);
+    return GetExecGraphInfoByPrimitivesInfo(primitives_info, false);
```

Comment:
Should we change the default behavior of this?

Reply:
It's just my mistake. It is restored to `true`. Thank you.